### DISCUSSION TITLE
Example for Next.js doesn't work (Error: Module not found: Can't resolve 'c-kzg')

### DISCUSSION THREAD

**Discussion by Igor-Koniukhov** [Web3Auth/Web3Auth#1374] [Help]
Hi. The sample from Using the Integration Builder block for Next.js doesn't work. There is a mistake in useEffect() - catch and init() mismatch with rows and doesn't visible for the rest code. And after compilation, I encountered with error - "./node_modules/@web3auth/ethereum-provider/node_modules/@ethereumjs/tx/dist/utils/blobHelpers.js:4:0
Module not found: Can't resolve 'c-kzg". Could you please hint, me how can I fix 'c-kzg' related error?

**Comment by samsoft00** [comment]
Hey, @Igor-Koniukhov encounter the same issue, I've been on it for hours.
I found the same issue here 👉  https://github.com/ethereumjs/ethereumjs-monorepo/issues/2551

Package Version
```json
"@web3auth/base": "^4.6.0",
"@web3auth/core": "^4.6.0",
"@web3auth/openlogin-adapter": "^4.6.0"
```

Any solution?
<img width="1016" alt="Screenshot 2023-02-25 at 8 43 47 AM" src="https://user-images.githubusercontent.com/5300891/221345377-a642b782-cfef-47b4-b2c3-341a2acca601.png">

@chaitanyapotti @arch1995 help.

**Reply by YZhenY** [reply]
This is unfortuantely an issue on the @etheruemjs side, we need them to adjust

**Reply by YZhenY** [reply]
try the fix stated here guys https://github.com/ethereumjs/ethereumjs-monorepo/issues/2551

**Reply by frankfka** [reply]
Seems like there's an upstream fix, once those are merged & released, could we update web3auth as well?

**Comment by frankfka** [comment]
Getting the same issue. Commenting to follow

**Comment by ninaYangAperture** [comment]
same above. waiting for solutions...

**Comment by nicolas-angelo** [comment]
Same issue

**Comment by KultureElectric** [comment]
Same Issue

**Comment by frankfka** [comment]
Any update on this?

**Comment by chaitanyapotti** [comment]
This is solved in v5
